enterprise/server/githubapp/runner.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
Can you please add a comment at the top about what this script does and why?

Reply:
Done